.github/workflows/delivery-docker.yml

```diff
@@ -0,0 +1,27 @@
+name: delivery / docker
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  deliver-docker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Determine version
+        id: version
+        shell: bash
+        run: |
+          echo "::set-output name=VERSION::$(echo ${{ github.event.release.tag_name }} | cut -d "v" -f2)"
+      - name: Build and Push Image
+        uses: docker/build-push-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+          repository: cnbs/pack
```

Comment:
Since this is a "prod" release it should be under the "buildpacksio" namespace/org. Eg. `buildpacksio/pack`